example/lib/pages/tabview_page.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flutter/cupertino.dart';
```

Comment:
Why the change to material from cupertino?

Reply:
My bad, it auto imported cupertino when I was tinkering with the Tabs widgets. Will fix.